app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -10,6 +10,7 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm < Man
   end
   supports :snapshots
   supports :snapshot_create
+  supports :revert_to_snapshot
```

Comment:
Instead of https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/331/files#diff-b0ec1540d8e7568e453fb09d326d45f1f9aea6af9fd06c4749fdcddd87c2a987R20 you could check the vm's power state here in a supports block, something like

```
supports :revert_to_snapshot do
  unsupported_reason_add(:revert_to_snapshot, "Cannot revert to snapshot while VM is running") unless power_state == "off"
end
```

Reply:
This is great as it grays out the "revert to snapshot" icon with a hover message, nullifying the need for check and worrying about duplicate notifications. Addressed in the newest force-push.